doc/examples/plot_line_hough_transform.py

```diff
@@ -64,70 +64,71 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
-# Construct test image
-
+# Constructing test image.
 image = np.zeros((100, 100))
-
-
-# Classic straight-line Hough transform
-
 idx = np.arange(25, 75)
 image[idx[::-1], idx] = 255
 image[idx, idx] = 255
 
+# Classic straight-line Hough transform.
 h, theta, d = hough_line(image)
 
-fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(8,4))
+# Generating figure 1.
+fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(12, 6))
```

Comment:
this is where @stefanv suggested to replace with ``(ax0, ax1, ax2)`` (since Python is counting from 0). Other occurrences below.